example/test/util/setup.js

```diff
@@ -0,0 +1,10 @@
+const server = require('../../server');
+const { setup: setupPuppeteer } = require('jest-environment-puppeteer')
```

Comment:
Oh, interesting, I thought `prettier` would make this consistent. I'll look into what happened, maybe it's not setup correctly.

Reply:
It seems `gts` doesn't run `prettier` on the example folder by default so I ran it manually. Potentially because the project in that folder is not TS.